### DISCUSSION TITLE
How to import models?

### DISCUSSION THREAD

**Discussion by DTPhilEng** [snipe/snipe-it#14432] [Q&A]
How do I import models in Snipe IT? I don't see that as an option when performing the standard import from the GUI

**Comment by ksgrg** [comment]
Hi

You can use the Import option to import assets with Model column on a CSV and it should import models in bulk.
You can download the samples from this page https://snipe-it.readme.io/docs/importing
![image](https://github.com/snipe/snipe-it/assets/162708626/3c93d67c-3503-44a7-a2eb-1cf7249c5de5)
